### PR TITLE
Update skeleton.md

### PR DESCRIPTION
**Pull request by obwansan**
Proposed changes to 'Create New Project' are to prevent frustration of having to create another skeleton app if you've made different selections at the start.

Proposed change to generate-factory-for-class is due to the command as presented not working for me. It only worked when I changed it to the format specified in https://docs.zendframework.com/zend-servicemanager/console-tools/#generate-factory-for-class.

Thanks
Kevin

Provide a narrative description of what you are trying to accomplish:

- [ ] Are you fixing a bug?
  - [ ] Detail how the bug is invoked currently.
  - [ ] Detail the original, incorrect behavior.
  - [ ] Detail the new, expected behavior.
  - [ ] Base your feature on the `master` branch, and submit against that branch.
  - [ ] Add a regression test that demonstrates the bug, and proves the fix.
  - [ ] Add a `CHANGELOG.md` entry for the fix.

- [ ] Are you creating a new feature?
  - [ ] Why is the new feature needed? What purpose does it serve?
  - [ ] How will users use the new feature?
  - [ ] Base your feature on the `develop` branch, and submit against that branch.
  - [ ] Add only one feature per pull request; split multiple features over multiple pull requests
  - [ ] Add tests for the new feature.
  - [ ] Add documentation for the new feature.
  - [ ] Add a `CHANGELOG.md` entry for the new feature.

- [ ] Is this related to quality assurance?
  

- [ ] Is this related to documentation?

### CODE REVIEW

File: docs/book/v2/getting-started/skeleton.md [hunk 1]
```diff
@@ -549,7 +550,7 @@ You should see the message change as you go between the two URIs!
 
 You likely don't want to hardcode HTML into your middleware; so, let's use
 templates. This particular exercise assumes you chose to use the
-[Plates](http://platesphp.com) integration.
+[Plates](http://platesphp.com) integration.
```
Comment (webimpress) [inline docs/book/v2/getting-started/skeleton.md]: Remove the space at the end of the line.

File: docs/book/v2/getting-started/skeleton.md [hunk 2]
```diff
@@ -615,13 +616,17 @@ For the next part of the example, we'll be creating and wiring a factory for
...
 
-zend-servicemanager provides a tool for generating factories based on
-reflecting a class; we'll use that to generate our factory:
+zend-servicemanager provides a console (command line) tool for generating factories 
+based on reflecting a class; we'll use that to generate our factory. Navigate to your
+root skeleton-app directory in the console and enter the following:
 
 ```bash
-$ ./vendor/bin/generate-factory-for-class "App\\Action\\HelloAction" > src/App/Action/HelloActionFactory.php
+$ ./vendor/bin/generate-factory-for-class \
```
Comment (froschdesign) [inline docs/book/v2/getting-started/skeleton.md]: Why was this line changed?

File: docs/book/v2/getting-started/skeleton.md [hunk 3]
```diff
@@ -615,13 +616,17 @@ For the next part of the example, we'll be creating and wiring a factory for
...
+> "App\\Action\\HelloAction" > ./src/App/src/Action/HelloActionFactory.php
 ```
 
+> For more information see [generate-factory-for-class](https://docs.zendframework.com/zend-servicemanager/console-tools/#generate-factory-for-class)
```
Comment (froschdesign) [inline docs/book/v2/getting-started/skeleton.md]: Adding the link is good and imporant, but the additional blockquote is not needed. Please add the link to the paragraph above.

File: docs/book/v2/getting-started/skeleton.md [hunk 4]
```diff
@@ -615,13 +616,17 @@ For the next part of the example, we'll be creating and wiring a factory for
 creating the `HelloAction` instance; the example assumes you used the default
 selection for a dependency injection container, zend-servicemanager.
 
-zend-servicemanager provides a tool for generating factories based on
-reflecting a class; we'll use that to generate our factory:
+zend-servicemanager provides a console (command line) tool for generating factories
```
Comment (froschdesign) [inline docs/book/v2/getting-started/skeleton.md]: The target audience of this documentation are developers and therefore "console tool" is enough.
(The link to the `generate-factory-for-class` command should be added here.)

**Comment by froschdesign** [comment]
@obwansan 
Is there a reason why you chose the tutorial for version 2 and not version 3 to work with zend-expressive?

File: docs/book/v2/getting-started/skeleton.md [hunk 5]
```diff
@@ -22,13 +22,14 @@ This will prompt you to choose:
   templates, configuration, assets, etc.).
 
 - A dependency injection container. We recommend using the default, Zend
-  ServiceManager.
+  ServiceManager. (We use ServiceManager in the examples below, so 
+  choose it if you want to code along).
```
Comment (xtreamwayz) [inline docs/book/v2/getting-started/skeleton.md]: This can be left out. The command below to generate  factory should be changed (more below).

File: docs/book/v2/getting-started/skeleton.md (hunk 1 above)
Comment (xtreamwayz) [inline docs/book/v2/getting-started/skeleton.md]: Instead of pinning to plates, why not supply template examples for twig and zend-view? Expressive is all about choices right?
